### PR TITLE
Docs: add how-to for Flatcar Container Linux

### PR DESCRIPTION
**Pull request by akurz**

### CODE REVIEW

File: docs/how-to/flatcar.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# How to Load DRBD on Flatcar Container Linux
+
+This guide shows you how to set up the DRBD® Module Loaderwhen using [Flatcar Container Linux](https://www.flatcar.org).
```
Comment (WanzenBug) [inline docs/how-to/flatcar.md:3]:
```suggestion
This guide shows you how to set up the DRBD® Module Loader when using [Flatcar Container Linux](https://www.flatcar.org).
```

File: docs/how-to/flatcar.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      volumes:
+        - name: usr-src
+          $patch: delete
+      initContainers:
+        - name: drbd-module-loader
+          volumeMounts:
+            - mountPath: /usr/src
+              name: usr-src
+              $patch: delete
+```
```
[the remark this answers is not in the document]
Comment (akurz) [reply docs/how-to/flatcar.md:34]: It works really nice on current Flatcar stable. All the header files/config files are available to build external modules for the running kernel. Building it within the jammy container works fine for us:
 
```Need a git checkout to regenerate drbd/.drbd_git_revision
make[1]: Entering directory '/tmp/pkg/drbd-9.2.7/drbd'

    Calling toplevel makefile of kernel source tree, which I believe is in
    KDIR=/lib/modules/6.1.77-flatcar/build

cat /dev/null   > Module.symvers
make -C /lib/modules/6.1.77-flatcar/build   M=/tmp/pkg/drbd-9.2.7/drbd  modules "PRE_CFLAGS="
warning: the compiler differs from the one used to build the kernel
  The kernel was built by: x86_64-cros-linux-gnu-gcc (Gentoo Hardened 13.2.1_p20230826 p7) 13.2.1 20230826
  You are using:           gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
....
make[1]: Leaving directory '/tmp/pkg/drbd-9.2.7/drbd'

	Module build was successful.
=======================================================================
  With DRBD module version 8.4.5, we split out the management tools
  into their own repository at https://github.com/LINBIT/drbd-utils
  (tarball at http://links.linbit.com/drbd-download)

  That started out as "drbd-utils version 8.9.0",
  has a different release cycle,
  and provides compatible drbdadm, drbdsetup and drbdmeta tools
  for DRBD module versions 8.3, 8.4 and 9.

  Again: to manage DRBD 9 kernel modules and above,
  you want drbd-utils >= 9.3 from above url.
=======================================================================

DRBD version loaded:
version: 9.2.7 (api:2/proto:86-122)
GIT-hash: 194036f7c69c45ab65549c6504c17523cb7d6644 build by @flatcar-02-00-0a-d2-78-26, 2024-02-29 08:49:12
Transports (api:20): tcp (9.2.7) lb-tcp (9.2.7) rdma (9.2.7)

File: docs/how-to/flatcar.md (hunk 1 above)
Comment (akurz) [reply docs/how-to/flatcar.md]: Updated, thx!

File: docs/how-to/flatcar.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (WanzenBug) [reply docs/how-to/flatcar.md:34]: Interesting, thanks!

**Review by WanzenBug — APPROVED** [review]

**Comment by akurz** [comment]
Sorry for the noise, another whitespace - I will close this PR and create a clean one.

**Pull request closed without merging**